content/en/docs/demos-examples/demos.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Demo Sites & Services"
+description: "Sites and services to demo passkeys"
+lead: "Sites and services to demo passkeys"
+date: 2023-09-19T16:45:00.148Z
+draft: false
+images: []
+menu:
+  docs:
+    parent: "demos-examples"
+weight: 1120
+toc: true
+---
+
+## General Passkey Demo Sites
+
+- [WebAuthn.io](https://webauthn.io)
```

Comment:
nit: I prefer to see the visible domain as "webauthn.io", something about "WebAuthn.io" is so weird to me lol
```suggestion
- [webauthn.io](https://webauthn.io)
```